packages/util/src/string.ts

```diff
@@ -0,0 +1,32 @@
+export const StringUtils = {
+  byteSize: (str: string) => new Blob([str]).size,
+  /**
+   * convert a string to an array of strings, items limited by the specified bytes
+   *
+   * @param str The string to slice up
+   * @param maxBytes number of bytes to impose as a limit
+   * @returns A sliced version of the str input
+   */
+  sliceByBytes: (str: string, maxBytes: number): string[] => {
```

Comment:
Naming suggestion: Lodash has a similar function and calls it '[chunk](https://lodash.com/docs/4.17.15#chunk)'
```suggestion
  chunkByBytes: (str: string, maxBytes: number): string[] => {
```

Reply:
Thanks! https://github.com/input-output-hk/cardano-js-sdk/pull/1427/commits/439491317951a2936c9c777b4eb87efda5abe228